core/prisma/exampleCommunitySeeds/unjournal.ts

```diff
@@ -288,101 +288,133 @@ export default async function main(prisma: PrismaClient, communityUUID: string)
 
 	await prisma.pubField.createMany({
 		data: [
-			{ id: fieldIds[0], name: "Title", slug: "unjournal:title", communityId: communityUUID },
+			{
+				id: fieldIds[0],
+				name: "Title",
+				slug: "unjournal:title",
+				communityId: communityUUID,
+				schemaName: CoreSchemaType.String,
+			},
 			{
 				id: fieldIds[1],
 				name: "Description",
 				slug: "unjournal:description",
 				communityId: communityUUID,
+				schemaName: CoreSchemaType.String,
```

Comment:
not really related to this PR, but our prod data now has schemaNames for unjournal, so just updating our seed data to match. not sure I chose the right schema for each type, but did my best!